src/Storages/FileLog/FileLogConsumer.cpp

```diff
@@ -155,21 +152,14 @@ void ReadBufferFromFileLog::readNewRecords(ReadBufferFromFileLog::Records & new_
     }
 }
 
-bool ReadBufferFromFileLog::nextImpl()
+ReadBufferPtr FileLogConsumer::getNextRecord()
 {
-    if (!allowed || !hasMorePolledRecords())
```

Comment:
`allowed` can probably be deleted now from this class, seems it is unused after this change